modules/tasks/tasks/src/main/resources/webapp/js/directives.js

```diff
@@ -681,7 +681,7 @@
                 var manipulationType = $scope.type.toLowerCase();
                 $scope.manipulationTypes = [];
                 if(['unicode', 'string'].indexOf(manipulationType) > -1) {
-                    $scope.manipulationTypes = ['toUpper', 'toLower', 'capitalize', 'URLEncode', 'join', 'split', 'substring', 'format', 'parseDate'];
+                    $scope.manipulationTypes = ['toUpper', 'toLower', 'capitalizeFirstLetter', 'URLEncode', 'join', 'split', 'substring', 'format', 'parseDate'];
```

Comment:
My question from the previous pull request remains. Does this work properly after this change? (This means, can I save the task with this manipulation, can I run such task successfully and are the resulting values really capitalized?)